pgt/src/PGT/Output/Definition/examples/failure-on-invalid-title.expected

```diff
@@ -0,0 +1,5 @@
+                                                example
```

Comment:
Comment: Given a DB object definition's name is a key arbiter in the definitions parser, it's therefore very compelling to add an example that asserts its importance.